python/example_code/neptune/tests/analytics_tests/test_execute_gremlin_profile_query.py

```diff
@@ -0,0 +1,80 @@
+import io
+import pytest
+from botocore.response import StreamingBody
+from botocore.exceptions import ClientError
+from analytics.neptune_analytics_query_example import run_open_cypher_query
+from neptune_graph_stubber import NeptuneGraphStubber
+
+GRAPH_ID = "test-graph-id"
+
+def test_execute_gremlin_profile_query(capfd):
+    stubber = NeptuneGraphStubber()
+    client = stubber.get_client()
+
+    # --- Success case with payload ---
+    stubber.activate()
+    payload_bytes = b'{"results": "some data"}'
+    response_body = StreamingBody(io.BytesIO(payload_bytes), len(payload_bytes))
+
+    stubber.stubber.add_response(
+        "execute_query",
+        {"payload": response_body},
+        {
+            "graphIdentifier": GRAPH_ID,
+            "queryString": "MATCH (n {code: 'ANC'}) RETURN n",
+            "language": "OPEN_CYPHER"
+        }
+    )
+    run_open_cypher_query(client, GRAPH_ID)
+    out, _ = capfd.readouterr()
+    assert '{"results": "some data"}' in out
+    stubber.deactivate()
+
+    # --- Success case with empty payload ---
+    stubber.activate()
+    empty_payload = StreamingBody(io.BytesIO(b''), 0)
+
+    stubber.stubber.add_response(
+        "execute_query",
+        {"payload": empty_payload},
+        {
+            "graphIdentifier": GRAPH_ID,
+            "queryString": "MATCH (n {code: 'ANC'}) RETURN n",
+            "language": "OPEN_CYPHER"
+        }
+    )
+    run_open_cypher_query(client, GRAPH_ID)
+    out, _ = capfd.readouterr()
+    assert out.strip() == ""  # Empty line
+    stubber.deactivate()
+
+    # --- ClientError case ---
+    stubber.activate()
+    stubber.stubber.add_client_error(
+        "execute_query",
+        service_error_code="ValidationException",  # <-- Updated to a valid exception code
```

Comment:
Can we remove all the comments that start with <-- ?

Reply:
Fixed

Reply:
I double checked all tests and comments are gone